Tests/PackageLoading/ConventionTests.swift

```diff
@@ -126,13 +441,20 @@ class ConventionTests: XCTestCase {
     }
 
     static var allTests = [
-        ("testDotFilesAreIgnored", testDotFilesAreIgnored),
-        ("testResolvesSingleSwiftModule", testResolvesSingleSwiftModule),
-        ("testResolvesSystemModulePackage", testResolvesSystemModulePackage),
-        ("testResolvesSingleClangModule", testResolvesSingleClangModule),
-        ("testMixedSources", testMixedSources),
-        ("testTwoModulesMixedLanguage", testTwoModulesMixedLanguage),
-        ("testCInTests", testCInTests),
+        ("testCInTests"                        , testCInTests),
```

Comment:
FWIW, I prefer we not generally use spaces for alignment unless it really adds clarity (which I'm not sure it does here).